src/components/Popover/Popover.scss

```diff
@@ -17,6 +17,7 @@ $content-max-width: rem(400px);
   will-change: opacity;
   opacity: 0;
   transition: opacity duration() easing(in);
+  transform: translate3d(0, 0, 0);
```

Comment:
What do you need this for?

Reply:
Was this left in from our different attempts?

Reply:
Yes this was left in for the debugging of the popover issue - I'll remove that